TagCloud.Domain/Layouters/TagCloudLayouter.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Drawing;
+using System.Numerics;
+
+public class TagCloudLayouter : ITagCloudLayouter
+{
+    private int i;
+    private List<RectangleF> rectangles = new List<RectangleF>();
+    private readonly PointF center;
+    private RectangleF currentRectangle;
+    private int segmentsCount = 2;
```

Comment:
```suggestion
    private int segmentCount = 2;
```

Также не хватает вводных данных почему тут магическое число?